### PR TITLE
Add the ability to create the first user

### PR DESCRIPTION
**Pull request by gssbzn**

### CODE REVIEW

File: cloudmanager/cloudmanager.go [hunk 1]
```diff
@@ -28,14 +28,12 @@ import (
...
-	Version = "0.1"
-	// DefaultBaseURL API default base URL for cloud manager
-	DefaultBaseURL = "https://cloud.mongodb.com" + APIPublicV1Path
-	// DefaultAPIPath default root path for all API endpoints
-	APIPublicV1Path = "/api/public/v1.0/"
-	// DefaultUserAgent To be submitted by the client
-	DefaultUserAgent = "pcgc/" + Version + " (" + runtime.GOOS + "; " + runtime.GOARCH + ")"
+	Version          = "0.1" // Version for client
+	CloudURL         = "https://cloud.mongodb.com"
+	DefaultBaseURL   = CloudURL + APIPublicV1Path                                            // DefaultBaseURL API default base URL for cloud manager
```
Comment (colm-quinn) [inline cloudmanager/cloudmanager.go:33]: Nice

File: cloudmanager/unauth_users.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 MongoDB Inc
```
Comment (colm-quinn) [inline cloudmanager/unauth_users.go]: [np] Consistency with other files where you used 2019-Present?

**Review by colm-quinn — APPROVED** [review]
LGTM. You're flying through these!

File: cloudmanager/unauth_users.go (hunk 2 above)
Comment (gssbzn) [reply cloudmanager/unauth_users.go]: copy/paste probably 😅

**Pull request merged**